libs/BLEKit/source/CoreGap.cpp

```diff
@@ -7,12 +7,12 @@ using namespace ble;
 
 void CoreGap::setDefaultAdvertising()
```

Comment:
question: instead of having two methods which do almost the same thing, `setDefaultAdvertising` the first time and then `refreshAdvertising`, why not just `refreshAdvertising(ble::advertising::default_data)`?